.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# VS Code
+.vscode/
```

Comment:
Just wondering, what's this for?  Usually `.vscode` is included

Reply:
I put a launch.json file for debugging in my local copy. Maybe this stuff is OK to include...

Reply:
That would be really useful to include

Reply:
OK I added back with sample launch config